# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/interfaces/moz.build
# Commit: 786805a2cdcc
# Full Hash: 786805a2cdcc4cdab632a955dace417062d0087e
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r effe12f2120a -r 786805a2cdcc accessible/interfaces/moz.build
--- a/accessible/interfaces/moz.build	Tue Nov 28 22:03:20 2023 +0000
+++ b/accessible/interfaces/moz.build	Tue Nov 28 22:03:21 2023 +0000
@@ -40,7 +40,6 @@
     "nsIAccessibleTextSelectionChangeEvent.idl",
     "nsIAccessibleTypes.idl",
     "nsIAccessibleValue.idl",
-    "nsIAccessibleVirtualCursorChangeEvent.idl",
 ]
 
 XPIDL_MODULE = "accessibility"